spec/queries/occupation_standard_query_spec.rb

```diff
@@ -140,4 +140,20 @@
 
     expect(occupation_standard_search.pluck(:id)).to contain_exactly(os1.id)
   end
+
+  it "allows searching by industry name" do
+    industry1 = create(:industry, name: "Healthcare Support Occupations")
+    industry2 = create(:industry, name: "Repair Occupations")
+
+    occupation_standard = create(:occupation_standard, industry: industry1)
+    create(:occupation_standard, industry: industry2)
+
+    params = {q: "Healthcare Support Occupations"}
```

Comment:
Would you mind changing this query to be something like: "healthcare support" so it also tests the case-insensitivity as well as the partial matching all at once? Otherwise, looks great!